### DISCUSSION TITLE
My builds get stuck in the "queued" state.

### DISCUSSION THREAD

**Discussion by ChampiMagic** [codemagic-ci-cd/codemagic-docs#1225] [Q&A] [answered]
![Captura de pantalla 2022-12-13 115247](https://user-images.githubusercontent.com/97173716/207366245-4a3cdc9d-ce0a-4a92-bd73-baf4fc167a54.png)

**Comment by K-Codemagic** [comment, marked as answer]
Hi, are you still facing this issue? If so, can you please share the build id?

**Reply by ChampiMagic** [reply]
Sorry for the delay in replying. No, after 50 minutes the builds were resumed.

**Reply by K-Codemagic** [reply]
It must be a temporary spike. You can always cancel a build if it is queued for a long time and start a new one. Sorry for the inconvenience caused.

**Comment by majirosstefan** [comment]
Same here. 

Several Android builds are not working last 1 - 1,5 hours. I have tried cancelling and creating new builds. Tried iOS build too, that one seemed to be working as expected.
Link to build is here: https://codemagic.io/app/634e9d09556598001385c88b/build/63c80f6e1e8443f8aaed722b

What should I do now, please ?  Just wait ? 

Also, it would be nice if there would be in-app notifications inside Codemagic console that would warn users about such "spikes".

**Reply by majirosstefan** [reply]
@K-Codemagic

**Reply by K-Codemagic** [reply]
Hi @majirosstefan it was an intermittent issue, it seems your latest build is successful.

**Comment by DreamakerDimas** [comment]
Hi @K-Codemagic
Same issue. 

Both android and iOS builds stuck in queued state. Build ID: 63d8f419c99fc8235bcc7aae

**Reply by DreamakerDimas** [reply]
Just started building. Thanks

**Reply by Mayb3Nots** [reply]
Having same issue too, is it normal to have such huge delays? Mine haven't start since few minutes has gone by.

**Reply by K-Codemagic** [reply]
Should be okay in a few minutes. Sorry for the inconvenience caused.

**Comment by idreesVenD** [comment]
Hi @K-Codemagic 
I am facing the same issue and it has been over an hour. Still waiting :(

**Reply by K-Codemagic** [reply]
We had a temporary issue with the availability of M1 build machines. This has been resolved now and all builds are expected to start soon. Appreciate your patience.

**Comment by leonganOax** [comment]
hi , my build also stuck in Status: queue , and it already have been 21 hours , the Id is 63feec62195e8d56cf8e506f , what can I do now?

**Reply by himesh-cm** [reply]
It should work ok now,
All builds should start shortly after the organic queue

**Comment by ktakayama** [comment]
Same here.
ID:63fef89d5bc56ed4b4f11249

And also ID:63fec1720225b3782c2fb000 is over 2 hours in Preparing build machine.

**Reply by ktakayama** [reply]
Just finished building. Thanks

**Comment by Bulu507** [comment]
I got the same issue
this is my ID: 63fef825a4d8d0233a291112

previously I've waited over an hour on id:63fef24e36465ed850f828ec. then I tried to cancel and try a new one but still stuck in the same issue.
